symengine/expression.h

```diff
@@ -120,12 +123,23 @@ class Expression
     {
         return eq(*m_basic, *other.m_basic);
     }
+
+    bool operator>=(const Expression &other) const
```

Comment:
I believe it's used in printer.h line 65. If removed, the compiler complains about it.